portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Gabi Newman's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Gabi Newman's Portfolio</h1>
+      <p>This is Gabi's portfolio.</p>
+      <p>Gabi is a junior at University of Massachusetts Amherst, where she is double majoring in Computer Science and Mathematics.
+        Is it weird that this page is written in the third person? Maybe. She's not sure, so let's just go with it.
+        Gabi's STEP internship in the summer of 2020 is her first time interning at Google, and she is very excited about all of the new things she hopes to learn!
+        When not doing CS, she enjoys ballroom/latin dancing, cross stitching, spending time with her friends (virtually, if she has to!).
+        She will be turning 20 years old in June.
```

Comment:
Break up long lines: go/htmlcssstyle#HTML_Line-Wrapping

Reply:
Done, sorry! I did this for the JS file but forgot to do it for this one. The style guide I have doesn't specify a column limit, so I just picked 80, since that's what it gives for JS.